vars/unitTestPost.groovy

```diff
@@ -44,46 +50,50 @@ def call(Map config = [:]) {
                      allowEmptyArchive: true
     return
   }
+  
+  def test_results = stage_info['junit_files']
+  junit testResults: test_results
 
-  junit testResults: config.get('testResults', 'test_results/*.xml')
-
-  def artifact_list = config.get('artifacts', ['run_test.sh/*', 'vm_test/**'])
+  def artifact_list = stage_info['artifacts']
```

Comment:
This needs to be reverted per the above comments in parseStageInfo.

parseStageInfo only provides defaults based on parsing the stage name.

For our current stage names that do not have information to be expected to be those settings become the default.

Reply:
I changed this, now I'm sending a value from daos Jenkinsfile and using config.get